README.md

```diff
@@ -1 +1,78 @@
+| * `ADAPTER`           | The adapter to use for traffic routing.                                                  | ""             | ""                                  |
+| `IPERF_PORT`          | Port to use for the iperf connection to the server. (`-p` arg).                          | "5201"         | "5201"                              |
+| `IPERF_DURATION`      | Duration of the iperf run in secs. (`-t` arg) (Multiple args delimited by `;`)           | "10"           | "10;60;300"                         |
+| `IPERF_PROTOCOL`      | Protocols to test (TCP and UDP). Accepts multiple args delimited by `;`.                 | "TCP"          | "TCP;UDP"                           |
+| `IPERF_DIRECTION`     | Direction of traffic to test. (Client to server, Reverse or bidirectional.               | "NORMAL"       | "NORMAL;REVERSE;BIDIRECTIONAL"      |
+| `IPERF_STREAMS`       | Streams to use for iperf run. (`-P` arg) (Multiple args delimited by `;`)                | "1"            | "1;2;4;8;16"                        |
+| `IPERF_BUFFER_LENGTH` | Buffer length for iperf TCP tests. (`-l` arg) (Multiple args delimited by `;`)           | "128k"         | "16k;128k;1M"                       |
+| `IPERF_WINDOW_SIZE`   | Buffer length for iperf TCP tests. (`-w` arg) (Multiple args delimited by `;`)           | Depends on OS  | "64k;128k;1M"                       |
+| `IPERF_MTU`           | Change the MTU size of the network. (`-M` arg). (Multiple args delimited by `;`)         | "1500"         | "576;1500;9000;1472;2000;3000;4000" |
+| `IPERF_LATENCY`       | Add custom latency to network.                                                           | "0s"           | "5ms;50ms;100ms"                    |
+| `IPERF_PACKET_LOSS`   | Add packet loss to network.                                                              | "0%"           | "0.5%;1%;3%"                        |
+| `PING_INTERVAL`       | Time interval in seconds between ping tests. (`-i` Arg) (Multiple args delimited by `;`) | "1"            | "0.1;0.5;1;5"                       |
+| `PING_PACKET_SIZE`    | Packet size of packets send by ping tests. (`-i` Arg) (Multiple args delimited by `;`)   | "56"           | "32;56;128;1024;4096"               |
+| `PING_LATENCY`        | Add custom latency to network.                                                           | "0ms"          | "5ms;50ms;100ms"                    |
+
+> EnvVars marked with a `*` are mandatory to run the tests.
+
+The `.env.sample` file contains the suggested variation values. Renaming it to `.env` will cause it to be loaded and 
+used by the script when it runs. Feel free to ignore it, if you are using another way to set EnvVars. For cases that no 
+configuration is provided the script will run the iperf and ping measurements using the default values.
+
+For the Name variable it is recommended to add a json formatted string to contain all the details of the hardware setup 
```

Comment:
I would suggest an example to clarify the Name variable. Maybe also indicate the issue of what happens when no Name is provided or Name is not altered on consecutive variation runs

Reply:
Added a sensible example.